### PR TITLE
Update country form field based on location.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: marketpulse/main/views.py [hunk 1]
```diff
@@ -20,6 +22,16 @@ def home(request):
 def edit_fxosprice(request, username='', id=None):
     user = request.user
 
+    country_code = ''
```
Comment (johngian) [inline marketpulse/main/views.py]: Please move this under `if lat and lang`. You could also initialize `country_code` with None which will be converted in js `null`.

File: marketpulse/main/static/js/location.js [hunk 2]
```diff
@@ -26,6 +26,23 @@ $(document).ready(function() {
...
+            'latitude': e.latlng.lat
+        };
+
+        $.ajax({
+            url: '/fxosprice/new/',
+            type: 'GET',
+            dataType: 'json',
+            data: data,
+            success: function(json) {
+                if (json.country !== null) {
```
Comment (johngian) [inline marketpulse/main/static/js/location.js:41]: Isn't `if json.country` enough?

**Comment by johngian** [comment]
LGTM r+wc :rabbit:

**Pull request merged**